README.md

```diff
@@ -113,7 +113,7 @@ we need to implement new features and help the community on a lot of issues.
 
 You can help us keep going and improving it by **[supporting the project](https://opencollective.com/emacs-lsp)**
 
-<img src="https://opencollective.com/emacs-lsp/tiers/backer.svg">
+<a href="https://opencollective.com/emacs-lsp"><img src="https://opencollective.com/emacs-lsp/tiers/backer.svg" /></a>
```

Comment:
Odd, for me the image already has a link on Github README

Reply:
Yes, but the point of this commit is to change it so that clicking on the image directs to the user to the donation page rather than github's hosted copy of the image.